### PR TITLE
Add RGB, RGBW and RGBWW capability to template.light

### PR DESCRIPTION
**Pull request by Vaarlion**
## Proposed change

This clean old mention of the white control that no-longer work since https://github.com/home-assistant/core/pull/76923 and add the new documentation for it's replacement, RGB, RGBW and RGBWW control.

It also replace the still working but deprecated `color` keyword for the new and more accurate `hs` keyword.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [X] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/86047
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/light.template.markdown [hunk 1]
```diff
@@ -165,12 +170,21 @@ light:
...
+        required: false
+        type: action
+      set_rgbw:
+        description: "Defines an action to run when the light is given a rgbw color command. Available variables: `rgbw` as a tuple, `rgb` as a tuple, `r`, `g`, `b` and `w`"
+        required: false
+        type: action
+      set_rgbww:
+        description: "Defines an action to run when the light is given a rgbww color command. Available variables: `rgbww` as a tuple, `rgb` as a tuple, `r`, `g`, `b`, `cw` and `ww`"
+        required: false
         required: false
```
Comment (YuriiMaiboroda) [inline source/_integrations/light.template.markdown:187]: `required: false` is duplicated

**Review by YuriiMaiboroda — CHANGES_REQUESTED** [review]
Need to remove duplicate of `required: false` for `set_rgbww`.

File: source/_integrations/light.template.markdown (hunk 1 above)
Comment (Vaarlion) [reply source/_integrations/light.template.markdown:187]: Good catch :)

**Review by YuriiMaiboroda — APPROVED** [review]
LGTM

**Comment by Vaarlion** [comment]
rebased one last time, i no longer working on this.
It's been to long.

File: source/_integrations/light.template.markdown [hunk 2]
```diff
@@ -105,13 +100,23 @@ light:
...
         type: template
         default: optimistic
-      white_value_template:
-        description: Defines a template to get the white value of the light.
+      hs_template:
+        description: Defines a template to get the hs color of the light. Must render a tuple (hue, saturation)
```
Comment (c0ffeeca7) [inline source/_integrations/light.template.markdown]:
```suggestion
        description: Defines a template to get the HS color of the light. Must render a tuple (hue, saturation).
```

File: source/_integrations/light.template.markdown [hunk 3]
```diff
@@ -105,13 +100,23 @@ light:
...
+        required: false
+        type: template
+        default: optimistic
+      rgb_template:
+        description: Defines a template to get the rgb color of the light. Must render a tuple or a list (red, green, blue)
```
Comment (c0ffeeca7) [inline source/_integrations/light.template.markdown]:
```suggestion
        description: Defines a template to get the RGB color of the light. Must render a tuple or a list (red, green, blue).
```

File: source/_integrations/light.template.markdown [hunk 4]
```diff
@@ -105,13 +100,23 @@ light:
...
+        required: false
+        type: template
+        default: optimistic
+      rgbw_template:
+        description: Defines a template to get the rgbw color of the light. Must render a tuple or a list (red, green, blue, white)
```
Comment (c0ffeeca7) [inline source/_integrations/light.template.markdown]:
```suggestion
        description: Defines a template to get the RGBW color of the light. Must render a tuple or a list (red, green, blue, white).
```

File: source/_integrations/light.template.markdown [hunk 5]
```diff
@@ -105,13 +100,23 @@ light:
...
         required: false
         type: template
         default: optimistic
-      color_template:
-        description: Defines a template to get the color of the light. Must render a tuple (hue, saturation)
+      rgbww_template:
+        description: Defines a template to get the rgbww color of the light. Must render a tuple or a list (red, green, blue, cold white, warm white)
```
Comment (c0ffeeca7) [inline source/_integrations/light.template.markdown]:
```suggestion
        description: Defines a template to get the RGBWW color of the light. Must render a tuple or a list (red, green, blue, cold white, warm white).
```

File: source/_integrations/light.template.markdown [hunk 6]
```diff
@@ -165,12 +170,20 @@ light:
...
-      set_white_value:
-        description: Defines an action to run when the light is given a white value command.
+      set_hs:
+        description: "Defines an action to run when the light is given a hs color command. Available variables: `hs` as a tuple, `h` and `s`"
         required: false
         type: action
-      set_color:
-        description: Defines an action to run when the light is given a color command.
+      set_rgb:
+        description: "Defines an action to run when the light is given a rgb color command. Available variables: `rgb` as a tuple, `r`, `g` and `b`"
```
Comment (c0ffeeca7) [inline source/_integrations/light.template.markdown]:
```suggestion
        description: "Defines an action to run when the light is given an RGB color command. Available variables: `rgb` as a tuple, `r`, `g` and `b`."
```

File: source/_integrations/light.template.markdown [hunk 7]
```diff
@@ -165,12 +170,20 @@ light:
...
+        required: false
+        type: action
+      set_rgbw:
+        description: "Defines an action to run when the light is given a rgbw color command. Available variables: `rgbw` as a tuple, `rgb` as a tuple, `r`, `g`, `b` and `w`"
```
Comment (c0ffeeca7) [inline source/_integrations/light.template.markdown]:
```suggestion
        description: "Defines an action to run when the light is given an RGBW color command. Available variables: `rgbw` as a tuple, `rgb` as a tuple, `r`, `g`, `b` and `w`."
```

File: source/_integrations/light.template.markdown [hunk 8]
```diff
@@ -165,12 +170,20 @@ light:
...
+        required: false
+        type: action
+      set_rgbww:
+        description: "Defines an action to run when the light is given a rgbww color command. Available variables: `rgbww` as a tuple, `rgb` as a tuple, `r`, `g`, `b`, `cw` and `ww`"
```
Comment (c0ffeeca7) [inline source/_integrations/light.template.markdown]:
```suggestion
        description: "Defines an action to run when the light is given an RGBWW color command. Available variables: `rgbww` as a tuple, `rgb` as a tuple, `r`, `g`, `b`, `cw` and `ww`."
```

File: source/_integrations/light.template.markdown [hunk 9]
```diff
@@ -194,8 +207,8 @@ For example, you would replace
 with this equivalent that returns `true`/`false` and never gives an unknown
 result:
 {% raw %}`{{ is_state('switch.source', 'on') }}`{% endraw %}
-Transition doesn't have its own script, it will instead be passed as a named parameter `transition` to the `turn_on`, `turn_off`, `brightness`, `color_temp`, `effect`, `hs_color` or `white_value` scripts.
-Brightness will be passed as a named parameter `brightness` to either of `turn_on`, `color_temp`, `effect`, `hs_color` or `white_value` scripts if the corresponding parameter is also in the call. In this case the brightness script (`set_level`) will not be called.
+Transition doesn't have its own script, it will instead be passed as a named parameter `transition` to the `turn_on`, `turn_off`, `brightness`, `color_temp`, `effect`, `hs_color`, `rgb_color`, `rgbw_color` or `rgbww_color` scripts.
+Brightness will be passed as a named parameter `brightness` to either of `turn_on`, `color_temp`, `effect`, `hs_color`, `rgb_color`, `rgbw_color` or `rgbww_color` scripts if the corresponding parameter is also in the call. In this case the brightness script (`set_level`) will not be called.
```
Comment (c0ffeeca7) [inline source/_integrations/light.template.markdown]:
```suggestion
Brightness will be passed as a named parameter `brightness` to either of `turn_on`, `color_temp`, `effect`, `hs_color`, `rgb_color`, `rgbw_color` or `rgbww_color` scripts if the corresponding parameter is also in the call. In this case, the brightness script (`set_level`) will not be called.
```

File: source/_integrations/light.template.markdown [hunk 10]
```diff
@@ -351,3 +364,58 @@ light:
 ```
 
 {% endraw %}
+
+### Make a Global light entity for a multi segments WLED light
```
Comment (c0ffeeca7) [inline source/_integrations/light.template.markdown]:
```suggestion
### Make a global light entity for a multi-segment WLED light
```

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @Vaarlion :+1:

**Review by c0ffeeca7 — APPROVED** [review]

**Pull request merged**